debian/build.sh

```diff
@@ -0,0 +1,194 @@
+#!/usr/bin/env bash
+
+trap 'umount "${MOUNT_DIR}" 2>/dev/null; rm -rf "${WORK_DIR}"' INT TERM EXIT
+
+# Prints a message in color
+function print_color() {
+    # Reset escape code
+    RST="\033[0m"
+    printf "\n%b%s%b\n" "${COLOR}" "${1}" "${RST}"
+}
+
+# Prints an error message in bold red then exits
+function die() {
+    COLOR="\033[01;31m" print_color "${1}"
```

Comment:
I'm surprised this works; can you just pass the color in as a normal parameter?

Reply:
Sure.